src/browser-realm.ts

```diff
@@ -118,7 +118,6 @@ function aggregateGlobalDescriptors(
 
     // removing unforgeable descriptors that cannot be installed
     delete to.location;
-    delete to.EventTarget;
```

Comment:
we were assuming that because its prototype was unforgeable, this global descriptor was non-configurable, but it is configurable and can be entirely replaced.